src/exchange/translations/Messages_fr_FR.xml

```diff
@@ -365,6 +366,7 @@
    <translation id="exchange_update_billing_header_1M" qtlid="8818">Mensuelle</translation>
    <translation id="exchange_update_billing_header_terminate" qtlid="209177">Aucune</translation>
    <translation id="exchange_update_billing_header_new_periode" qtlid="190992">Nouvelle période</translation>
+   <translation id="exchange_update_billing_periode_action">Modifier les périodes de renouvellement en</translation>
```

Comment:
```suggestion
   <translation id="exchange_update_billing_periode_monthly">Modifier les périodes de renouvellement en mensuelle</translation>
   <translation id="exchange_update_billing_periode_monthly">Modifier les périodes de renouvellement en annuelle</translation>
```
This is better of the translators :)

Also, the wording is weird

Reply:
Ok, do you have another suggestion for the wording ?

Reply:
No 😅

Reply:
Thanks for you suggestion :)
This will be modified in a shorter way like `Modifier en "Mensuelle"`. "Mensuelle" is a value.

Reply:
`Passer tous les services en renouvellement mensuel`
`Passer tous les services en renouvellement annuel`
?